detox/test/e2e/02.matchers.test.js

```diff
@@ -4,19 +4,21 @@ describe('Matchers', () => {
     await element(by.text('Matchers')).tap();
   });
 
-  it('should match elements by (accesibility) label', async () => {
+  it('should match elements by (accessibility) label', async () => {
     await element(by.label('Label')).tap();
     await expect(element(by.text('Label Working!!!'))).toBeVisible();
   });
 
-  it('should match elements by (accesibility) id', async () => {
+  it('should match elements by (accessibility) id', async () => {
     await element(by.id('UniqueId345')).tap();
     await expect(element(by.text('ID Working!!!'))).toBeVisible();
   });
 
   it('should match elements by index', async () => {
     const index = device.getPlatform() === 'ios' ? 2 : 0;
-    await element(by.text('Index')).atIndex(index).tap();
+    await element(by.text('Index'))
+      .atIndex(index)
```

Comment:
Any reason to break this into multiple lines? Please revert this change.

Reply:
That's a changed done by Prettier. I can revert it.

Reply:
Please revert. Thanks